server/src/data/listings.ts

```diff
@@ -12,6 +14,45 @@ export const getListing = async (listingId: string) => {
   return listing.data();
 };
 
+export const createListing = async (title: String, description: String, price: Number, street: String, city: String, state: String, zip: String, lat: number, lon: number, ownerId: String, imageArray) => {
+  const docRef = await firestore.addDoc(collection, {
+    title: title,
+    description: description,
+    price: parseFloat(price.toFixed(2)),
+    ownerId: ownerId,
```

Comment:
I think we have been calling it owner and not ownerId in firestore.

Reply:
The "bookings" collection has "ownerId", is there anywhere else that says just "owner"? Just want to confirm before I make the change.

Reply:
oh your right, my bad. Just got confused since listings has just owner.